docs/basso-livello/medical.md

```diff
@@ -0,0 +1,206 @@
+---
+template: home.html
+title: Sistema Medico
+---
+
+# Sistema Medico
+
+Questa sezione copre la standardizzazione del sistema medico usato dal Network Bagigi.
+
+Le informazioni in questa guida fanno riferimento alle documentazioni delle mod [ACE Medical](https://ace3.acemod.org/wiki/feature/medical-system) e [KAT Advanced Medical](https://docs.google.com/document/d/1YlY00qoL2gdrbOJJzi5aW19A3HqT7uXflszDYuK7h3M/edit).
+
+## Concetti generali
+
+A prescindere dalle mod e i settaggi esatti in uso, esistono dei concetti generali utili a medicare altri giocatori in missione in maniera efficiente.
+
+### Il personale medico
+
+#### Il Fante
+
+Un fante senza specializzazione può a volte stabilizzare un ferito senza richiedere l'intervento di un soccorritore/medico, ma anche nei casi in cui non gli è possibile sarà comunque in grado di alleggerire il compito al personale medico.
+
+Per questo è importante che ogni fante conosca le fondamenta del sistema medico e settaggi in uso, al fine di accorciare il più possibile i tempi di cura (propri e altrui) e snellire il flusso della missione.
+
+#### Il Soccorritore
+
+Un "mini-medico" a livello di gruppo/squadra che potrà spesso stabilizzare in autonomia pazienti lievi, ma contare sul supporto del medico per casi gravi.
```

Comment:
Ma contare?

Reply:
Cosa suonerebbe meglio secondo te?

Reply:
Un "mini-medico" all'interno del gruppo/squadra, capace di stabilizzare autonomamente pazienti con condizioni lievi, ma in grado di fare affidamento sul supporto del medico per i casi più gravi.

Reply:
> all'interno del gruppo/squadra

Non saprei, visto che anche nella sezione dopo del medico si riferisce di più al livello gerarchico dell'unità, al posto di esserne solo "all'interno".

Il resto ok.